radixdlt-core/radixdlt/src/integration/java/com/radixdlt/integration/steady_state/simulation/full_function_forks/CoordinatedForkSanityTest.java

```diff
@@ -190,7 +191,7 @@ public void sanity_tests_should_pass() {
                     // verify that at epoch 21 we've successfully switched to the fork with voting
                     // (idx 3)
                     if (!verifyCurrentFork(network, forks.get(3))) {
-                      reportError.accept("Expected to be at a different fork (3) at epoch 20");
+                      reportError.accept("Expected to be at a different fork (3) at epoch 21");
```

Comment:
Was it a bug in the code or the comment was wrong?

Reply:
the error message was wrong

Reply:
maybe it would be better to replace the pair of numbers by variables.

Reply:
I'm not sure, there are 5 different epochs where the test does things, I'd need to add vars for all of them